src/rqt_top/top_plugin.py

```diff
@@ -51,14 +51,14 @@ def __lt__(self, other):
 
 class Top(Plugin):
 
-    NODE_FIELDS   = [             'pid', 'get_cpu_percent', 'get_memory_percent', 'get_num_threads']
-    OUT_FIELDS    = ['node_name', 'pid', 'cpu_percent',     'memory_percent',     'num_threads'    ]
-    FORMAT_STRS   = ['%s',        '%s',  '%0.2f',           '%0.2f',              '%s'             ]
-    NODE_LABELS   = ['Node',      'PID', 'CPU %',           'Mem %',              'Num Threads'    ]
-    SORT_TYPE     = [str,         str,   float,             float,                float            ]
-    TOOLTIPS      = {
+    NODE_FIELDS = ['pid', 'get_cpu_percent', 'get_memory_percent', 'get_num_threads']
```

Comment:
I think the white space helps formatting here

Reply:
I like it too but I'm not sure about selective linting with flake8 in the ROS2 buildfarm. I think you are right that we should leave it for now.